libsmtutil/Z3Interface.cpp

```diff
@@ -194,7 +194,7 @@ z3::expr Z3Interface::toZ3Expr(Expression const& _expr)
 		}
 		else if (n == "tuple_get")
 		{
-			size_t index = std::stoi(_expr.arguments[1].name);
+			size_t index = std::stoul(_expr.arguments[1].name);
```

Comment:
I'm fine with this, but I think we should be consistent. The fix above (in the GH diff) casts `stoi` to `size_t`.

Reply:
I would prefer this type of solution with `stoul` without casting. Will update other two occurrences